src/poetry/utils/password_manager.py

```diff
@@ -173,6 +174,25 @@ def set_pypi_token(self, name: str, token: str) -> None:
             self.keyring.set_password(name, "__token__", token)
 
     def get_pypi_token(self, name: str) -> str | None:
+        """Get PYPI token.
+
+        First checks the enviroment variables for a token,
+        then the configured username/password and last the
+        available keyring.
+
+
+        Parameters
+        ----------
+        name
+            PYPI username.
```

Comment:
this parameter is the repository name, isn't it, not the user name?  https://github.com/python-poetry/poetry/blob/master/src/poetry/publishing/publisher.py#L59

Reply:
Im not sure, on discord I mentioned that Im confused about what the `name` means. Not gonna lie most names here are very vague

Reply:
It is the repo name

Reply:
@Darsstar changed, please check